core/input/input.cpp

```diff
@@ -270,6 +271,15 @@ bool Input::is_mouse_button_pressed(MouseButton p_button) const {
 	return mouse_button_mask.has_flag(mouse_button_to_mask(p_button));
 }
 
+bool Input::_config_ignore_joybad_on_unfocused_window() {
+	if (bool(GLOBAL_GET("input_devices/config/ignore_joypad_on_unfocused_window"))) {
```

Comment:
I think this should be cached in a member variable so it doesn't need to be read from the config each time there's an event.

Then the setting can either be made to require a restart (`GLOBAL_DEF_RST`), or config changes can be handled via the `ProjectSettings.settings_changed` signal.

Reply:
Thx for the info about the signal, i will make the suggested changes.